rpcs3/Emu/RSX/Overlays/overlay_trophy_notification.cpp

```diff
@@ -34,17 +34,20 @@ namespace rsx
 
 		trophy_notification::trophy_notification()
 		{
-			frame.set_pos(0, 0);
-			frame.set_size(300, 80);
-			frame.back_color.a = 0.85f;
-
-			image.set_pos(8, 8);
-			image.set_size(64, 64);
+			frame.set_pos(68.24, 55.34);
+			frame.set_size(425, 72);
+			frame.back_color.r = 0.247059f;
+			frame.back_color.g = 0.250980f;
+			frame.back_color.b = 0.247059f;
+			frame.back_color.a = 0.88f;
+
+			image.set_pos(78, 64);
+			image.set_size(53.333, 53.333);
 			image.back_color.a = 0.f;
 
-			text_view.set_pos(85, 0);
-			text_view.set_padding(0, 0, 24, 0);
-			text_view.set_font("Arial", 9);
+			text_view.set_pos(139.14, 69.30);
```

Comment:
the parameters you pass are doubles, but set_pos only accepts u16 (a.k.a. no decimal places)